### PR TITLE
validation worker - queries snapshot app and looks for changes.  Repo…

### PR DESCRIPTION
**Pull request by LMWF**
# Description

This is the validation worker described in the link below.  This queries the snapshot app and looks for changes.  It reports metrics on call times and failure rates, including when stats from the snapshot app haven't changed between 2 queries.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #16

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: validation-worker/.gitignore [hunk 1]
```diff
@@ -0,0 +1,329 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (amanbha) [inline validation-worker/.gitignore:1]: It can be moved to the root of the repo.

Comment (LMWF) [reply validation-worker/.gitignore]: Deleted, already a version at root.

File: validation-worker/validation-worker.csproj [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Dapr.AspNetCore" Version="0.5.0-preview02" />
```
Comment (amanbha) [inline validation-worker/validation-worker.csproj:8]: this can use 0.6.0-preview01

Comment (LMWF) [reply validation-worker/validation-worker.csproj:8]: Updated that and the other refs.

**Pull request merged**